src/Microsoft.Azure.NotificationHubs/XiaomiNotification.cs

```diff
@@ -0,0 +1,74 @@
+﻿//----------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved. 
+// Licensed under the MIT License. See License.txt in the project root for 
+// license information.
+//----------------------------------------------------------------
+
+using System;
+using System.Text;
+
+namespace Microsoft.Azure.NotificationHubs
+{
+    /// <summary>
+    /// Represents a Xiaomi notification.
+    /// </summary>
+    public sealed class XiaomiNotification : Notification, INativeNotification
+    {
+        static string contentType = "application/json";
+        static string targetPipeline = "X-Target-Pipeline";
+        static string serviceBusNotificationFormat = "ServiceBusNotification-Format";
+        static string newPipeline = "New";
```

Comment:
Should this be here? For testing purposes, but probably should have it configurable at some point